drydock/templates/kustomized/tutor16/extensions/hpa.yml

```diff
@@ -0,0 +1,82 @@
+---
+apiVersion: autoscaling/v1
```

Comment:
Should we move to the V2 of the [HPA api](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#horizontalpodautoscaler-v2-autoscaling)?

Reply:
Currently V2 is failing due to https://github.com/argoproj/argo-cd/issues/9145 in out ArgoCD version, it shows an error when is deployed and shows unknow status. In ArgoCD Stage is working fine. @jfavellar90

Reply:
To prevent blocking this, just move forward with V1